static/app/components/actions/resolve.tsx

```diff
@@ -134,61 +132,62 @@ class ResolveActions extends React.Component<Props> {
       ? t('Set up release tracking in order to use this feature.')
       : '';
 
-    const actionLinkProps = {
-      shouldConfirm,
-      message: confirmMessage,
-      confirmLabel,
-      disabled: disabled || !hasRelease,
+    const onActionOrConfirm = onAction => {
+      if (shouldConfirm) {
```

Comment:
You can get rid of these `if/else` by passing the [`bypass` parameter](https://github.com/getsentry/sentry/blob/9ffb055577cce118fb51042273bc57b90b73329f/static/app/components/confirm.tsx#L133) to `openConfirmModal`.

Reply:
Ahh that's much nicer!